helm-chart/ray-cluster/README.md

```diff
@@ -34,15 +34,15 @@ kubectl get pods
 # raycluster-kuberay-worker-workergroup-2jckt   1/1     Running   0          66s
 
 # Step 6: Forward the port of Dashboard
-kubectl port-forward --address 0.0.0.0 svc/raycluster-kuberay-head-svc 8265:8265
```

Comment:
`0.0.0.0` is necessary in some cases. For example, if I want to access the port 8265 in my devbox via the `$DEVBOX_IP:8265`, I need to set `0.0.0.0`.

Reply:
Yeah totally understand there might be cases where something like this might be appropriate, but its probably not the right default since most users run `kubectl` on the device they're going to be connecting from.

Also our dev box setup isn't putting that on the public internet, but other user setups may accidentally do so.

Reply:
> since most users run kubectl on the device they're going to be connecting from.

I don't think so. Most users use Macs with ARM chips, but Ray/KubeRay is not friendly to ARM chips.

Reply:
> Also while our dev box setup isn't putting that on the public internet, but other user setups may accidentally do so.

We can add comments mentioning the security concerns. However, I think we should still keep `0.0.0.0`. It's not necessary to include `0.0.0.0` in the default command, but we at least need to mention it in the comments.

Reply:
I was about to say the same too... in practice most dev/test would want 0.0.0.0 just to expose the Ray Dashboard quickly so you can get up and running which was the aim of this example.